_slides/12-pierremartindumont.md

```diff
@@ -3,7 +3,8 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+This file was hidden on my file system, but finally got it!
+Well, my bad, it was under the _Slides folder. 
```

Comment:
Comment from KW

Reply:
hey Keith, thanks for providing a 2nd opinion on my edits.